addons/website/data/website_data.xml

```diff
@@ -120,19 +120,19 @@
             <field name="is_published">True</field>
             <field name="url">/</field>
             <field name="view_id" ref="homepage"/>
-            <field name="is_tracked">True</field>
+            <field name="track">True</field>
```

Comment:
Whyyy ? No ! Boolean field should start is_ or has_ and 'is_tracked' is way more easy to grep than 'track'.

Reply:
Ha because track is back on view.